src/types/callable/function.js

```diff
@@ -33,11 +33,10 @@ export function createFunction(info) {
       throw createGError(error[0]);
     }
 
-    const retVal = unboxArgument(returnType, returnValue[0]);
+    const retVal = unboxArgument(returnType, returnValue);
```

Comment:
would it make sense to make this block re-usable so it can be also used in `callable/method.js` and `callable/vfunc.js`, maybe as a return of `parseCallableArgs`